data_steward/tools/consolidated_achilles_report/get_all_results_html.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
```

Comment:
This is a completely new file, but I don't see any associated unit tests.  We should build our unit tests as we add our code.  Otherwise, we won't know when we've made a breaking change.  There should be at least one test method for each public function in this module.  Ideally, there will be multiple unit tests for each function:  a minimum of one "happy-path" test and other tests for possible bad inputs and expected behaviours.

Reply:
you are right, this was created in very short time and I was only looking if it works. I will try to make it a single script.